.github/workflows/main.yml

```diff
@@ -18,9 +18,7 @@ jobs:
         with:
           java-version: '8'
           distribution: 'adopt'
-      - name: Downloading docker-compose files
-        run: wget https://raw.githubusercontent.com/authorjapps/zerocode-docker-factory/master/compose/kafka-schema-registry.yml
       - name: Running Kafka
-        run: docker-compose -f kafka-schema-registry.yml up -d && sleep 10
+        run: docker-compose -f docker/compose/kafka-schema-registry.yml up -d && sleep 10
```

Comment:
@authorjapps  hey these are the changes to run the docker compose file present in the repo. What are your thoughts on these changes?

Reply:
> @authorjapps hey these are the changes to run the docker compose file present in the repo. What are your thoughts on these changes?

@a1shadows , Looks good to me. Did you check whether CI picked this command?

Reply:
@authorjapps  Yes, as per the CI logs, can see this ran successfully:
```
Run docker-compose -f docker/compose/kafka-schema-registry.yml up -d && sleep 10
```

can see the init container also started in the logs:
```
Creating compose_init-kafka-container_1 ... done
```